docs/developer-docs/latest/developer-resources/cli/CLI.md

```diff
@@ -98,6 +98,58 @@ strapi watch-admin
 options: [--browser <name>]
 ```
 
+
+## strapi export <BetaBadge />
+
+Exports your project data. The default settings create a `.tar` file, compressed using `gzip` and encrypted using `aes-128-ecb`.
+
+```bash
+strapi export
+```
+
+The exported file is automatically named using the format `export_YYYYMMDDHHMMSS` with the current date and timestamp. Alternately, you can specify the filename using the `-f` or `--file` flag. The following table provides all of the available options as command line flags:
```

Comment:
I believe the current behaviour is to overwrite an existing file if its name is given, but I think that will be changed before release to require the filename not to already exist. Not sure how to handle that in documentation, but wanted to point it out.

Reply:
Just wanted to check if I should change this for the release tomorrow?